sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -2084,10 +2084,9 @@ private void SetNameFieldsIfNull()
 
         #region UploadBlob
         /// <summary>
-        /// The <see cref="UploadBlob"/> operation creates a new block
-        /// blob or updates the content of an existing block blob in this
-        /// container.  Updating an existing block blob overwrites any existing
-        /// metadata on the blob.
+        /// The <see cref="UploadBlobAsync"/> operation creates a new block
```

Comment:
Should be UploadBlob rather than UploadBlobAsync.

Reply:
In the spirit of helping users solve the problem that started this change, should we also mention that users can call `GetBlobClient()` and then pass specialized options into `UploadAsync` if they need more fine grained control?

Reply:
Fixed, and added more detail.